docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -72,3 +72,18 @@ items:
       kubernetes.io/hostname: ocp-worker-0
       node-role.kubernetes.io/worker: ""
 ```
+
+// TODO: Move this to the IPv6 section once it is fully documented, both upstream and downstream.
+ifeval::["{build}" != "downstream"]
+[WARNING]
+In IPv6, {rhocp_long} worker nodes need a `/64` prefix allocation due to OVN
+limitations (RFC 4291). For dynamic IPv6 configuration, you need to change the
+prefix allocation on the Router Advertisement settings. If you want to use
+manual configuration for IPv6, define a similar CR to the
+`NodeNetworkConfigurationPolicy` CR example in this procedure, and define an
+IPv6 address and disable IPv4.  Because the constraint for the `/64` prefix did
+not exist in {OpenstackPreviousInstaller}, your previous {OpenStackShort}
+control plane network might not have enough capacity to allocate these
```

Comment:
```suggestion
`NodeNetworkConfigurationPolicy` CR example in this procedure, and define an
IPv6 address and disable IPv4. Because the constraint for the `/64` prefix did
not exist in {OpenstackPreviousInstaller}, your {OpenStackShort}
control plane network might not have enough capacity to allocate these
```

Reply:
Just some additional minor edits.

Reply:
Done, thanks